recipes/openldap/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "openldap" %}
+{% set version = "2.4.46" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-{{ version }}.tgz
+  sha256: 9a90dcb86b99ae790ccab93b7585a31fbcbeec8c94bf0f7ab0ca0a87ea0c4b2d
+
+build:
+  number: 0
+  skip: True  # [not unix]
+  run_exports:
+    - {{ pin_subpackage('openldap', max_pin='x.x') }}
+  script:
+    # disable server components
```

Comment:
Agreed that _most_ folks will just be looking for clients, but the merits of having a compatible `slapd` are considerable, especially for a testing case: if you can't make your integration work against a version built from exactly the same source, good luck with 389 or keycloak or (shudder) AD.

But indeed this gets us into libdb territory...

Reply:
oh and all the weird `libexec` stuff, and the schemas, and...

Reply:
We can build the server components as another output later on.  But that's future work

Reply:
yeah and i'd like to also avoid needing to pull in berkerley db for this